maintainers/maintainer-list.nix

```diff
@@ -5860,6 +5860,12 @@
     githubId = 57304299;
     keys = [ { fingerprint = "33CD 5C0A 673C C54D 661E  5E4C 0DB5 361B EEE5 30AB"; } ];
   };
+  dopplerian = {
```

Comment:
Changes to the maintainer list should be in a separate commit, preceeding the one adding the package entitled `maintainers: add dopplreian`

Reply:
Thank you a lot for all the feedback!

How should I go about doing these changes? Do I just redo the changes in two commits and then force-push?

Reply:
Yes, you can just start again and remake
- save the files somewher
- `git checkout --hard HEAD^` would eliminate those commits
- add the files one by one, remaking commits
- force push

Reply:
Done, it should be correct now.

Thank you for your time.